### DISCUSSION TITLE
database migration taking longer than 40+ minutes, m1 macbook air

### DISCUSSION THREAD

**Discussion by JosephChaaban** [ppy/osu#18492] [Q&A] [answered]
Hello,

I've downloaded lazer for apple sillicon and the first thing it does when I launch it is start database migration, fair enough. Except it has been going on for more than 40 minutes, and apparently that's not supposed to happen according to other people who use lazer, and laptop is heating up during the process. I have around 8 to 10k maps in the stable osu on my device. Should I just wait, or is this likely another issue?

**Comment by peppy** [comment, marked as answer]
It should show you a message telling you that you need to run migration using the non-m1 version. Can't really tell why it isn't as you haven't provided logs or anything, but you can try to run the intel version to recover.

**Reply by JosephChaaban** [reply]
Hello, and thank you for the reply!

It did show me this message, but I assumed it was referring to the stable release of osu (which is technically intel) and it made sense to me, considering it needs to migrate from somewhere after all.
So I just need to install the intel version of lazer?

**Reply by JosephChaaban** [reply]
Follow up: Sure enough, I did that and it ran in an instant. Thanks again, and sorry for that!

**Reply by peppy** [reply]
great to hear
